styles.css

```diff
@@ -482,3 +492,285 @@ textarea {
   width: 100%;
   margin: 124px 0 24px 0;
 }
+
+@media (min-width: 768px) {
```

Comment:
- [ ] Great job implementing the media queries 👏  but according to the requirements, the desktop width should be in 992px - 1100px range. You should consider adjusting the `min-width`, and don't forget to double-check the elements to see if they still match the figma design.

![image](https://user-images.githubusercontent.com/66967274/144540038-a7e7187a-e90e-431f-a3b1-3ef8ae355441.png)